ui/lib/kubernetes/addon/components/page/configure.js

```diff
@@ -0,0 +1,77 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { task } from 'ember-concurrency';
+import { waitFor } from '@ember/test-waiters';
+import errorMessage from 'vault/utils/error-message';
+
+export default class ConfigurePageComponent extends Component {
+  @service router;
+  @service store;
+
+  @tracked inferredState;
+  @tracked modelValidations;
+  @tracked error;
+  @tracked showConfirm;
+
+  constructor() {
+    super(...arguments);
+    if (!this.args.model.isNew && !this.args.model.disableLocalCaJwt) {
+      this.inferredState = 'success';
+    }
+  }
+
+  get isDisabled() {
+    if (!this.args.model.disableLocalCaJwt && this.inferredState !== 'success') {
+      return true;
+    }
+    return this.save.isRunning || this.fetchInferred.isRunning;
+  }
+
+  leave(route) {
+    this.router.transitionTo(`vault.cluster.secrets.backend.kubernetes.${route}`);
+  }
+
+  @action
+  onRadioSelect(value) {
+    this.args.model.disableLocalCaJwt = value;
+    this.inferredState = null;
+  }
+
+  @task
+  @waitFor
+  *fetchInferred() {
+    try {
+      yield this.store.adapterFor('kubernetes/config').checkConfigVars(this.args.model.backend);
```

Comment:
Probably not necessary if you're the only one working on implementation for this feature - but might be worth making a note that this function is mocking a WIP endpoint?

Reply:
I'm not sure it's necessary to know that in the component but I think it is a good idea to point it out in the adapter so I will add that. Thanks!